### ISSUE TITLE
Trouble with development

### ISSUE THREAD

**Issue by SpencerMycek** [NosVladimir/KineticArchetypes#14]
Hello!

I forked this repo, just to poke at some ideas I’ve had with vital strike/vital blade infusion. And I’ve run into a very tiny issue.
Easily able to get it to compile/build/clean, everything is very nicely set up.
But when trying to test it in game, it appears I’m having issues with localization. 
any string that comes from KineticArchetypes (even if I haven’t changed anything) all appear in game as “{ unknown key: x }”

Have you encountered this or have any tips?
I believe it’s just not reading the json correctly, or having troubles finding it, but final mod folder structure and VS look fine

**Comment by NosVladimir** [comment]
Hi Spence, no I haven't encountered this before, that's strange. Just to double check, I don't use anything other than Wolfie's BPCore, so if you have additional packages/libraries you can try to disable them first and see what happens?

I'm just about to go sleep now, will try to look into this tomorrow

**Comment by SpencerMycek** [comment]
No worries
I have done some testing, and noticed with your release dll, it only fails if I use the localization file I added to
And with my dll, it only fails with the localization file I added to

Which means I have somehow done something to make BPCore not like my json. Something for me to figure out

**Comment by SpencerMycek** [comment]
No, it was my fault. In my copy+paste haste, I had a key that was duplicated. And it was causing issues
